meson.build

```diff
@@ -45,12 +45,19 @@ endif
 
 sd_journal = dependency('libsystemd-journal', required : false)
 if not sd_journal.found()
-	sd_journal = dependency('libsystemd', required : false)
+  sd_journal = dependency('libsystemd', required : false)
 endif
 if sd_journal.found()
 	add_project_arguments('-DUSE_JOURNALD=1', language : 'c')
 endif
 
+if run_command('./hack/seccomp-notify.sh', check: false).returncode() == 0
+  seccomp = dependency('libseccomp', required : false)
```

Comment:
You don't seem to ever add this dependency anywhere?

Reply:
fun fact: I've never used meson before! thanks for the heads up :-)